extensions/tldr/package.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tldr",
+  "title": "tldr",
+  "author": "pomdtr",
+  "license":"MIT",
+  "description": "Search tldr Pages",
+  "icon": "tldr.png",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search TLDR Pages",
+      "subtitle": "tldr",
+      "description": "Search Commands Cheatsheets.",
+      "mode": "view"
+    },
+    {
+      "name": "refresh-pages",
+      "title": "Refresh TLDR Pages",
+      "subtitle": "tldr",
+      "description": "Refresh TLDR Pages",
```

```suggestion
      "description": "Refresh tldr Pages",
```

Should be lowercase to match the rest